maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -55,6 +57,7 @@ class AndroidDriver(
         .usePlaintext()
         .build()
     private val blockingStub = MaestroDriverGrpc.newBlockingStub(channel)
+    private val blockingStubWithTimeout get() = blockingStub.withDeadlineAfter(40, TimeUnit.SECONDS)
```

Comment:
Why 40 seconds? 

Do we want to use blockingStubWithTimeout for every call now?

Reply:
> Why 40 seconds?

I'll make more tests on this number, but don't expect a great ordeal out if this. I think we need just a high number to avoid the client waiting indefinitely.

> Do we want to use blockingStubWithTimeout for every call now?

Either this or use the blockingStub with a specific Deadline. The important here is that using a blockingStub without a Deadline means that the call will wait forever for a response - which makes no sense.

Reply:
> Either this or use the blockingStub with a specific Deadline. The important here is that using a blockingStub without a Deadline means that the call will wait forever for a response - which makes no sense.

Yup sounds good

> I'll make more tests on this number, but don't expect a great ordeal out if this. I think we need just a high number to avoid the client waiting indefinitely.

No issues, aligned on this now! nit: if its possible to log if when time is expired on the maestro side that will be useful.

Reply:
It will throw a very specific exception, so this will lead to a problem if any